src/main/webapp/script.js

```diff
@@ -158,46 +238,6 @@ function addAutoResize() {
   });
 }
 
-
-/**
- * Displays navbar authentication and inbox buttons according to login status.
- */
-function fetchAuthentication() {
-  fetch('/authentication').then(response => response.json()).then(user => {
-    const inboxButton = document.getElementById("notificationsDropdown");
-    if (user.isUserLoggedIn) {
-      // If user is logged in, show logout and inbox buttons in navbar.
-      inboxButton.style.display = "block";
-      loadNotifications(); 
-      if (!user.isUserRegistered) {
-        // If logged in user is not registered, redirect to signup page.
-        window.location.replace(user.authenticationUrl);
-      }
-      // Delete signup button.
-      const signupButtonNavbar = document.getElementById('signup');
-      signupButtonNavbar.innerHTML = '';
-
-      // Add logout button to navbar.
-      addAuthenticationButton(
-          user.authenticationUrl, 'btn-outline-success', 'Log Out', 'login');
-
-      // Show question submission box.
-      const questionSubmission = document.getElementById('post-question');
-      questionSubmission.style.display = "block";
-    } else {
-      // If user is logged out, show signup and login buttons in navbar.
-
-      // Add signup button to navbar.
-      addAuthenticationButton(
-          user.authenticationUrl, 'btn-success', 'Sign Up', 'signup');
-
-      // Add login button to navbar.
-      addAuthenticationButton(
-          user.authenticationUrl, 'btn-outline-success', 'Log In', 'login');
-    }
-  })
-}
-
 /**
  * Creates a signup, login, or logout button and appends it to navbar.
```

Comment:
I know this right here is my function, but could you rename it to `createAuthenticationButton()'. Again, to be consistent with function names and what they serve for (and again, would need reordering to where create functions are located).

Reply:
set a TODO

Reply:
nvm lol